Dockerfile.e2e

```diff
@@ -1,5 +1,7 @@
 FROM node:10
 
+RUN curl -sSL https://get.docker.com/ | sh
```

Comment:
Let's wait for final decision in https://github.com/poanetwork/tokenbridge/pull/191. I provided there few options how we can avoid docker-in-docker environment.